overlay/source/elm_overlayframe.hpp

```diff
@@ -8,7 +8,7 @@
  * @brief The base frame which can contain another view
  *
  */
-class SysTuneOverlayFrame : public tsl::elm::Element {
+class SysTuneOverlayFrame final : public tsl::elm::Element {
```

Comment:
Nice catch. I think it's technically irrelevant since we're not referencing the class type directly so the compiler isn't allowed to optimize function calls.

Reply:
That's correct, and gcc doesn't suggest labelling it final either. Was really a style change (I should probably stop doing them)